libs/compute/include/hpx/compute/vector.hpp

```diff
@@ -168,11 +168,13 @@ namespace hpx { namespace compute {
         {
             if (this == &other)
                 return *this;
+            
+            allocator_type tmp_alloc = other.alloc_;
```

Comment:
Couldn't this copy of the allocator be used below (https://github.com/STEllAR-GROUP/hpx/pull/4326/files#diff-008c8c59bedcccf092e71b5aac998cc6R187) when assigning to `alloc_`? I.e. something like:
```
    alloc_ = std::move(tmp_alloc);
```

Reply:
Good point! Updated the PR.